spec/brainstem/query_strategies/paginator_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+describe Brainstem::QueryStrategies::Paginator do
+  using_mysql = ActiveRecord::Base.connection.instance_values["config"][:adapter] =~ /mysql/i
+  if using_mysql
+    xit '- Tests requiring sqlite have not been run.' do
+      expect(true).to eq false
```

Comment:
Could we possibly raise here instead of a true == false assertion? If we raise an error and give a good error message perhaps we can help anyone who runs into a failure here.

Reply:
Yeah, I like that a lot!